dds_cli/__main__.py

```diff
@@ -649,17 +644,90 @@ def get(
         sys.exit(1)
 
 
+# COMMAND##########################################################################
+#### AUTH ################################################################## AUTH #
 ###################################################################################
-# CREATE ################################################################# CREATE #
-###################################################################################
-@dds_main.command(no_args_is_help=True)
+@dds_main.group()
+@click.pass_obj
+def auth(_):
+    """Manage the saved authentication token."""
+
+
+@auth.command()
 @click.option(
     "--username",
     "-u",
     required=False,
     type=str,
-    help="Your Data Delivery System username.",
+    help="Your Data Delivery System username. Required unless the `--check` flag is used.",
```

Comment:
What does the `--check` flag do? Is this something new you're implementing?

Reply:
No, it was already there. I just reordered the auth commands, didn't add anything.

Reply:
Oh ok. Then git is doing something strange since it's saying that it's added in this PR. But anyway, then you can remove this, we don't know what this is. 👍🏻

Reply:
Done!